samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatController.cs

```diff
@@ -89,6 +95,13 @@ public ChatController(ILogger<ChatController> logger)
             return this.NotFound($"Failed to find {ChatSkillName}/{ChatFunctionName} on server");
         }
 
+        // Broadcast bot typing state to all users
+        if (ask.Variables.Where(v => v.Key == "chatId").Any())
+        {
+            var chatId = ask.Variables.Where(v => v.Key == "chatId").First().Value;
```

Comment:
Can there be multiple chatIds in the context variables? How do you know the first one is right?

Reply:
We should catch these kinds of errors if there are duplicating variables in the Ask.

Reply:
Can be done in another PR

Reply:
documentation updates can also be done in another PR, just make sure you're tracking an issue for these things